outlines/models/transformers.py

```diff
@@ -185,11 +185,18 @@ def __init__(
             the `transformers` API for tokenizers.
 
         """
+        from transformers import FlaxPreTrainedModel
```

Comment:
It looks like this requires `flax` to be installed :/ Let's see if we can do this another way.

Reply:
Can't we just add flax to the test dependencies? We already have jax there

Reply:
It will still fail whenever users try to initialise a `transformers` model. We could use `try/except` for the checks (also add `TFPretrainedModel`).

Reply:
Ah yes, right

Reply:
I've added a `TensorAdapter` for `tensorflow`